frontend/src/contexts/StudentContext.ts

```diff
@@ -0,0 +1,19 @@
+import { createContext } from "react";
+
+import { TestResponse } from "../APIClients/types/TestClientTypes";
+
+type StudentContextType = {
+  test: TestResponse | null;
```

Comment:
for now, the StudentContext stores the whole test object, even though there are some unused values. figured it's better than defining more types, since that's getting a bit messy! but lmk if you have different thoughts